public/css/components/footer.css

```diff
@@ -0,0 +1,76 @@
+.footer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+
+    & > .container {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 4.9rem 0;
+    }
+
```

Comment:
BEM methodology doesn't recommend to use nested selectors [https://en.bem.info/methodology/css/#nested-selectors](https://en.bem.info/methodology/css/#nested-selectors)